docs/resources/domain_record.md

```diff
@@ -0,0 +1,176 @@
+---
+page_title: "Scaleway: scaleway_domain_record"
+description: |-
+  Manages Scaleway Domain records.
+---
+
+# scaleway_domain_record
+
+Creates and manages Scaleway Domain record.  
+For more information, see [the documentation](https://www.scaleway.com/en/docs/scaleway-dns/).
+
+## Examples
+
+### Basic
+
+```hcl
+resource "scaleway_domain_record" "www" {
+  dns_zone = "domain.tld"
+  name     = "www"
+  type     = "A"
+  data     = "1.2.3.4"
+  ttl      = 3600
+}
+resource "scaleway_domain_record" "www2" {
```

Comment:
Could you jump a line between the resources? Easier to copy and paste isolated examples. You can also have little section with reminder of the differences between the main record type

Reply:
```suggestion

resource "scaleway_domain_record" "www2" {
```